zilliqa/src/consensus.rs

```diff
@@ -1446,6 +1442,7 @@ impl Consensus {
         };
 
         // At genesis it could be fine not to have a qc block, so don't error.
+        // TODO: this should be an explicit check for genesis
```

Comment:
Raise an issue?

Reply:
Actually on looking further, this isn't a hole in block validation, it just skips finalizing the block if we have insufficient data to do so. Since this can now happen not only on genesis but also when loading from a checkpoint, I've just updated the comments to be more clear; no issue necessary.